website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -54,12 +59,36 @@ models:
         granularity: day # set granularity at column-level for standard_granularity_column
 ```
 </File>
+</VersionBlock>
 
-For an example project, refer to our [Jaffle shop](https://github.com/dbt-labs/jaffle-sl-template/blob/main/models/marts/_models.yml) example.
+<VersionBlock lastVersion="1.8">
+<File name="models/_models.yml">
+  
+```yaml
+models:
+  - name: time_spine_hourly
+    description: "my favorite time spine"
+    time_spine:
+      standard_granularity_column: date_hour # column for the standard grain of your table
+    columns:
+      - name: date_hour
+        granularity: hour # set granularity at column-level for standard_granularity_column
+  - name: time_spine_daily
+    time_spine:
+      standard_granularity_column: date_day # column for the standard grain of your table
+    columns:
+      - name: date_day
+        granularity: day # set granularity at column-level for standard_granularity_column
+```
+</File>
+</VersionBlock>
 
-Now, break down the configuration above. It's pointing to a model called `time_spine_daily`. It sets the time spine configurations under the `time_spine` key. The `standard_granularity_column` is the lowest grain of the table, in this case, it's hourly. It needs to reference a column defined under the columns key, in this case, `date_hour`. Use the `standard_granularity_column` as the join key for the time spine table when joining tables in MetricFlow. Here, the granularity of the `standard_granularity_column` is set at the column level, in this case, `hour`.
+For an example project, refer to our [Jaffle shop](https://github.com/dbt-labs/jaffle-sl-template/blob/main/models/marts/_models.yml) example.
 
+#### Understanding time-spine and granularity
+The previous configuration demonstrates a time-spine model called `time_spine_daily`. It sets the time spine configurations under the `time_spine` key. The `standard_granularity_column` is the lowest grain of the table, in this case, it's hourly. It needs to reference a column defined under the columns key, in this case, `date_hour`. Use the `standard_granularity_column` as the join key for the time spine table when joining tables in MetricFlow. Here, the granularity of the `standard_granularity_column` is set at the column level, in this case, `hour`. Additionally, the `custom_granularities` field lets you specify non-standard time periods like `fiscal_year` or `retail_month` that your organization may use.
```

Comment:
Instead of:
"The `standard_granularity_column` is the lowest grain of the table, in this case, it's hourly."
Can we say:
The `standard_granularity_column` is the column that maps to one of our standard granularities [list or link to options here]. The grain of this column must be finer or equal in size to the granularity of all custom granularity columns in the same model. In this case, it's hourly.
Slightly more accurate!

Reply:
not: when referencing the `columns` key, can we put that in quotes like the other keys referenced?

Reply:
Instead of:
"Use the `standard_granularity_column` as the join key for the time spine table when joining tables in MetricFlow."
"MetricFlow will use the `standard_granularity_column` as the join key when joining the time spine table to other source table."

Reply:
And one more thing - can we gate anything that references custom granularities on version 1.9+? Or add a note about upgrading like you did in the custom calendar section.